### DISCUSSION TITLE
Welcome to azerothcore-transmog-3.3.5a Discussions!

### DISCUSSION THREAD

**Discussion by DanieltheDeveloper** [DanieltheDeveloper/azerothcore-transmog-3.3.5a#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mx29rtd** [comment]
Any way to open the transmog pannel from a macro?

**Reply by DanieltheDeveloper** [reply]
Hey @Mx29rtd ,
I think that's something we could definitely add, since currently there isn't a way to open it via chat or macro. Feel free to submit a feature request on the discussion page!

https://github.com/DanieltheDeveloper/azerothcore-transmog-3.3.5a/discussions/14

**Comment by SQWIZI1337** [comment]
Hi. I updated your project a little, in case you need it.

1. I made a localization for the Russian language.
2. I made it so that when you put an item in your bag and take it out again, the transmogification doesn't reset

[azerothcore-transmog-3.3.5a.zip](https://github.com/user-attachments/files/24995678/azerothcore-transmog-3.3.5a.zip)
